### PR TITLE
Feature/voxel voronoi

### PR DESCRIPTION
**Pull request by jgostick**
Opening PR for @TomTranter to get his changes on to master

### CODE REVIEW

**Comment by bspellio** [comment]
@jgostick @TomTranter You guys should give this another look over before we merge.

**Comment by jgostick** [comment]
Wow, that is a lot of code!  Nothing jumps out at me, and I trust that @TomTranter knows the coding style of OpenPNM, so I say let's do it!

**Comment by bspellio** [comment]
capture

**Comment by TomTranter** [comment]
Yeah sorry it took so long for me to wrap this all up. I can test everything once merged on Monday, then I'm off to efcf

Sent from my Sony Xperia™ smartphone


---- jgostick wrote ----


Wow, that is a lot of code! Nothing jumps out at me, and I trust that @TomTranter<https://github.com/TomTranter> knows the coding style of OpenPNM, so I say let's do it!

File: OpenPNM/Geometry/__Toray120__.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+class Toray120(GenericGeometry):
+    r"""
+    Toray120 subclass of GenericGeometry
+    """
+
+    def __init__(self, **kwargs):
+        r"""
+        Initialize
+        """
+        super(Toray120, self).__init__(**kwargs)
```
Comment (bspellio) [inline OpenPNM/Geometry/__Toray120__.py]: Let's use python3's super: 
```python
super().__init__(**kwargs)
```

**Comment by TomTranter** [comment]
@jgostick This is ready to be merged in now. Looks like you'll have to do it manually... not sure why

**Pull request merged**